src/filesystem.rs

```diff
@@ -34,12 +36,6 @@ bitflags! {
     }
 }
 
-impl FileSystemFlags {
-    pub const fn const_empty() -> FileSystemFlags {
```

Comment:
While this is now uncovered by tests, it was used by static-filesystem, so I think we should keep it.

Reply:
It's not needed anymore, `bitflags` now just has `const` methods.